pages/index.js

```diff
@@ -1,18 +1,35 @@
-import React from 'react';
```

Comment:
rename file to HomeQuarto.js

Reply:
Not such a good idea, Next.js is waiting this file as a default page. 
And rename it will force to rewrite all routing rules while next does it very well itself.

Reply:
Ok then